auth-service/src/main/kotlin/uk/gov/communities/delta/auth/utils/CSVUtil.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.communities.delta.auth.utils
+
+private val csvChars = Regex("[,\"]")
+
+fun StringBuilder.csvRow(row: List<String>) {
+    for (i in row.indices) {
+        val element = row[i]
+        if (element.contains(csvChars)) {
+            append('"')
+            append(element.replace("\"", "\"\""))
+            append('"')
+            if (i < row.size - 1) append(',')
+        } else {
+            append(element)
+            if (i < row.size - 1) append(',')
+        }
+    }
+    append('\n')
```

Comment:
I looked for a library for this, but they mostly take objects and I've added dynamic columns for the actionData attributes, then it didn't seem worth pulling in a library for a 20 line helper function